pkg/flare/envvars.go

```diff
@@ -87,6 +87,11 @@ var allowedEnvvarNames = []string{
 
 	// CI
 	"DD_INSIDE_CI",
+
+	// Cluster agent
+	HELM_CHART_RELEASE_NAME,
```

Comment:
Why the Namespace envvar is not added here?

Reply:
As discussed in DM, the variable `DD_KUBE_RESOURCES_NAMESPACE` is already present in the flare, as it's in the Datadog config : https://github.com/DataDog/datadog-agent/blob/82493e20361640b3f50057b617f07e831931081e/pkg/config/config.go#L707